pytorch_lightning/trainer/training_loop.py

```diff
@@ -59,8 +60,8 @@ def on_trainer_init(self, max_epochs, min_epochs, max_steps, min_steps, num_sani
         self.trainer.train_dataloader = None
         self.automatic_optimization = automatic_optimization
 
-        self.trainer.max_epochs = max_epochs
-        self.trainer.min_epochs = min_epochs
+        self.trainer.max_epochs = 1000 if (max_epochs is None and max_steps is None) else max_epochs
```

Comment:
This seems to be the super hacky solution @ananthsub mentioned in the issue. Can we instead use the more clean one that also accepts None here and simply does not break the loop?

Reply:
Apologies if I'm misunderstanding since I'm not very familiar with the code base--my intention for this was to set `max_epochs` to 1000 if neither `max_epochs` nor `max_steps` were specified. If only `max_steps` is specified, then `max_epochs` will be None and the iterator [here](https://github.com/PyTorchLightning/pytorch-lightning/pull/4086/files#diff-0cd7170e96efa4a55577086f5db2fe4cR475-R476) will handle that appropriately.

Reply:
@richardqiu do you mind adding this as a comment in the code, so folks reading this will see that 1000 is the existing default for `max_epochs`?